tests/conftest.py

```diff
@@ -349,6 +350,13 @@ def application_docker_image(
     runtime: str,
     application_image_tag: str,
 ) -> Iterable[Image]:
+    if platform.machine() == "aarch64":
```

Comment:
Can you move all `platform.machine() == "aarch64"` and `platform.machine() != "aarch64"` checks to a `is_aarch64` function it tests' utils, and use it?